live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -231,6 +231,23 @@
           name: {{ lookup('env', 'APPLIANCE_USERNAME') }}
           lock_passwd: false
 
+#
+# In order application processes to generate core dumps, we need to
```

Comment:
missing "for" -- In order "for" applications

Reply:
Thanks! Fixed.